Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -192,6 +192,51 @@ def test_clipboard_astral(self):
         with self.assertRaises(tkinter.TclError):
             root.clipboard_get()
 
+    def test_mainloop_dispatching(self):
+        # reconstruct default root destroyed by AbstractTkTest
+        root = tkinter._default_root = self.root
+        for thing in (root.tk, root, tkinter):
```

Comment:
```suggestion
        for obj in (root.tk, root, tkinter):
```
These are all technically objects, and I feel `thing` isn't appropriate